src/components/Home/StepperSection.tsx

```diff
@@ -0,0 +1,356 @@
+import React from 'react';
+
+//utils
+import styled from 'styled-components';
+
+//icons
+import GoogleIcon from '../../assets/icons/Google.png';
+import FacebookIcon from '../../assets/icons/Facebook.png';
+
+//mui
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepLabel from '@mui/material/StepLabel';
+import StepContent from '@mui/material/StepContent';
+import Button from '@mui/material/Button';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+//components
+import DropZone from '../DropZone';
+
+const StyledContainer = styled(Box)`
+  padding: 0 9.438rem 5.688rem 10.063rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 2.5rem;
+`;
+
+const Title = styled(Typography)`
+  padding-bottom: 1.5rem;
+`;
+
+const VideoSection = styled.section`
+  padding-top: 1.125rem;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  gap: 3.875rem;
+  width: 70.5rem;
+  height: 21.063rem;
+`;
+
+const DropZoneSection = styled.section`
+  padding-top: 2.125rem;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 2.125rem;
+
+  width: 70.25rem;
+  height:auto;
+`;
+
+const IconContainer = styled(Box)`
+  position: relative;
+  width: 100%;
+`;
+
+const Icon = styled.img`
+  position: absolute;
+  top: 1.05rem;
+  left: 12.55rem;
+  z-index: 1;
+  width: 3.125rem;
+  height: 3.125rem;
+`;
+
+function getSteps() {
+  return [
+    <Typography
+      sx={{
+        lineHeight: '2.5rem',
+        fontWeight: 600,
+        fontSize: '1.728rem',      
+      }}
+      component="b"
+    >
+      Let's get started by requesting your data
+    </Typography>,
+    <Typography
+      id="dropzone"
+      sx={{
+        lineHeight: '2.5rem',
+        fontWeight: 600,
+        fontSize: '1.728rem',
+   
+      }}
+      component="b"
+    >
+      Now, you just have to upload the data files
```

Comment:
Ici, pour remplacer "upload", j'aurais mis : "Now, you just have to convert the data files".

Reply:
Done